src/middleware/externalRequests.js

```diff
@@ -419,8 +419,8 @@ const extractParamValue = (path, ctx) => {
       return extractValueFromObject(ctx.state.allData.state, path)
     case 'lookupRequests':
       return extractValueFromObject(ctx.state.allData.lookupRequests, path)
-    case 'responseBody':
-      return extractValueFromObject(ctx.state.allData.responseBody, path)
+    case 'requestBody':
```

Comment:
Hey man, good work. We already have a case for extracting from the request body, that is the `"payload"` case. To retrieve from the request body you use a path with the prefix `payload.`. This property `ctx.state.allData.requestBody` is an object that will contain the data after mapping, hence the name responseBody.

Reply:
:+1: Fixed - the test touching this was wrong hence I thought it needed changing.

Why is `payload` different from the others? This is what caused the confusion. 
But I realise it can't easily be changed at this point without affecting a lot of stuff

Reply:
Awesome, so I don't know the reason why that "case" was added because it's not necessary. I do not see a case where we extract a value from the mapped object. I think Ryan thought that this is where the request body will be when he worked on the logic for extracting from 'ctx.state' to populate the external requests urls. I think we should actually remove this case.

Reply:
I think we might have a few naming inconsistencies which could be causing some of this confusion. Might be a big job to rethink/rename some of these properties but def needs a re-look in the future.